modules/ossm-deploying-bookinfo-application.adoc

```diff
@@ -0,0 +1,83 @@
+:_mod-docs-content-type: Procedure
+[id="deploying-book-info"]
+= Deploying the Bookinfo application
+:context: ossm-pdeploying-book-info
+
+.Prerequisites
+
+* You have deployed a cluster on {ocp-product-title} 4.15 or later.
+* You are logged in to the {ocp-product-title} web console as a user with the `cluster-admin` role.
+* You have access to the {ocp-short-name} CLI (oc).
+* You have installed the {SMProductName} Operator, and deployed {istio}.
+* You have created the {istio} and IstioCNI resources.
+
+. In the {ocp-product-title} web console, navigate to the *Home* -> *Projects* page.
+
+. Click *Create Project*.
+
+. Enter `bookinfo` in the *Project name* field.
++
+The *Display name* and *Description" fields provide supplementary information and are not required. 
+
+. Click *Create*.
+
+. Apply the the {Istio} injection label to the `bookinfo` namespace by entering the following command at the command line:
++
+[source,terminal]
+----
+$ oc label namespace bookinfo istio-injection=enabled
+----
+
+. Apply the `bookinfo` YAML file to deploy the `bookinfo` application by entering the following command at the CLI:
++
+[source,terminal]
+----
+oc apply -f https://raw.githubusercontent.com/istio/istio/release-1.21/samples/bookinfo/platform/kube/bookinfo.yaml -n bookinfo
```

Comment:
Do we want to use a release branch (1.21) or use `master` like what the gateway samples use?
```suggestion
oc apply -f https://raw.githubusercontent.com/istio/istio/master/samples/bookinfo/platform/kube/bookinfo.yaml -n bookinfo
```

Reply:
done